src/utils/hooks/api/loans/use-get-loan-assets.tsx

```diff
@@ -0,0 +1,24 @@
+import { LoanAsset, TickerToData } from '@interlay/interbtc-api';
+import { useErrorHandler } from 'react-error-boundary';
+import { useQuery } from 'react-query';
+
+interface LoanAssetsData {
+  assets: TickerToData<LoanAsset> | undefined;
+}
+
+const getLoanAssets = (): Promise<TickerToData<LoanAsset>> => {
+  return window.bridge.loans.getLoanAssets();
+};
+
+const useGetLoanAssets = (): LoanAssetsData => {
+  const { data: assets, error } = useQuery({
```

Comment:
Shouldnt the loading status be taken into consideration?

Reply:
It is taken into consideration implicitly - when data are being loaded, the returned value is `undefined` - therefore we do not need to return extra value.